### PR TITLE
Updating changes

### PR DESCRIPTION
**Pull request by sherrif10**
Fixes #{issue number here}

Changes proposed in this pull request:
- 
- 
-

### CODE REVIEW

File: docs/docs/manual/cellediting.md [hunk 1]
```diff
@@ -103,52 +103,94 @@ You can also export the currently identified clusters as a JSON file, or close t
...
+    * [Beider-Morse](#baider-morse)
+- [Nearest Neighbor](#nearest-neighbor)
+    * [Levenshtein](#levenshtein-distance)
+    * [PPM](#ppm)
+
+#### Key Collision {#key-collision}
 
 **Key collisions** are very fast and can process millions of cells in seconds:
 
-**Fingerprinting** is the least likely to produce false positives, so it’s a good place to start. It does the same kind of data-cleaning behind the scenes that you might think to do manually: fix whitespace into single spaces, put all uppercase letters into lowercase, discard punctuation, remove diacritics (e.g. accents) from characters, split up all strings (words) and sort them alphabetically (so “Zhenyi, Wang” becomes “wang zhenyi”).
```
Comment (sherrif10) [inline docs/docs/manual/cellediting.md:125]: This description that fingerprinting does is awesome.

**Comment by sherrif10** [comment]
Hello @wetneb  Hope you are tracking this pull request.

**Comment by wetneb** [comment]
I am not because you have not opened it in the official repository but in your fork.

**Comment by sherrif10** [comment]
This need to be closed . Sorry.

**Pull request closed without merging**